datasets/plugins/batch/hive_dataset.py

```diff
@@ -176,8 +179,13 @@ def write_spark(
                 tmp_partition_by += ["run_time"]
 
         df, partition_cols = self._write_data_frame_prep(df, partition_by=tmp_partition_by)
+
         _logger.info(f"write_spark({self.hive_table=}, {partition_cols=})")
+        _retry_with_backoff(partial(self._write_spark_helper, spark, df, partition_cols, **kwargs))
```

Comment:
do you want to add a comment or TODO about  the retry here so we remember why this is being used?

Reply:
done!